.gitignore

```
@@ -0,0 +1 @@
+seasonal/evergreen/.DS_Store
```

```suggestion
.DS_Store
```
This will make it apply to all `.DS_Store`s, instead of just the ones in the specified folder.